fuel-android/build.gradle

```diff
@@ -32,11 +29,6 @@ android {
         }
     }
 
-    compileOptions {
```

Comment:
I think we have added this to accommodate this issue #116. I am all in to get rid of this but would it be a problem for our users?